src/Common/src/System/Drawing/ColorTable.cs

```diff
@@ -10,26 +10,52 @@ namespace System.Drawing
-            FillConstants(dict, typeof(SystemColors));
+            // Because there's no SystemColors, but they do persist in NamedColors, we need to manually add them here.
+            // Colors from a default XP desktop for use by UI designers in ASP.NET
+            dict.Add("ActiveBorder", Color.FromArgb(unchecked((int)unchecked((int)0xffd4d0c8)));
+            dict.Add("ActiveCaption", Color.FromArgb(unchecked((int)0xff0054e3)));
+            dict.Add("ActiveCaptionText", Color.FromArgb(unchecked((int)0xffffffff)));
+            dict.Add("AppWorkspace", Color.FromArgb(unchecked((int)0xff808080)));
+            dict.Add("ButtonFace", Color.FromArgb(unchecked((int)0x0)));
+            dict.Add("ButtonHighlight", Color.FromArgb(unchecked((int)0x0)));
+            dict.Add("ButtonShadow", Color.FromArgb(unchecked((int)0x0)));
+            dict.Add("Control", Color.FromArgb(unchecked((int)0xffece9d8)));
+            dict.Add("ControlDark", Color.FromArgb(unchecked((int)0xffaca899)));
+            dict.Add("ControlDarkDark", Color.FromArgb(unchecked((int)0xff716f64)));
+            dict.Add("ControlLight", Color.FromArgb(unchecked((int)0xfff1efe2)));
+            dict.Add("ControlLightLight", Color.FromArgb(unchecked((int)0xffffffff)));
+            dict.Add("ControlText", Color.FromArgb(unchecked((int)0xff000000)));
+            dict.Add("Desktop", Color.FromArgb(unchecked((int)0xff004e98)));
+            dict.Add("GradientActiveCaption", Color.FromArgb(unchecked((int)0x0)));
+            dict.Add("GradientInactiveCaption", Color.FromArgb(unchecked((int)0x0)));
+            dict.Add("GrayText", Color.FromArgb(unchecked((int)0xffaca899)));
+            dict.Add("Highlight", Color.FromArgb(unchecked((int)0xff316ac5)));
+            dict.Add("HighlightText", Color.FromArgb(unchecked((int)0xffffffff)));
+            dict.Add("HotTrack", Color.FromArgb(unchecked((int)0xff000080)));
+            dict.Add("InactiveBorder", Color.FromArgb(unchecked((int)0xffd4d0c8)));
+            dict.Add("InactiveCaption", Color.FromArgb(unchecked((int)0xff7a96df)));
+            dict.Add("InactiveCaptionText", Color.FromArgb(unchecked((int)0xffd8e4f8)));
+            dict.Add("Info", Color.FromArgb(unchecked((int)0xffffffe1)));
+            dict.Add("InfoText", Color.FromArgb(unchecked((int)0xff000000)));
+            dict.Add("Menu", Color.FromArgb(unchecked((int)0xffffffff)));
+            dict.Add("MenuBar", Color.FromArgb(unchecked((int)0x0)));
+            dict.Add("MenuHighlight", Color.FromArgb(unchecked((int)0x0)));
+            dict.Add("MenuText", Color.FromArgb(unchecked((int)0xff000000)));
+            dict.Add("ScrollBar", Color.FromArgb(unchecked((int)0xffd4d0c8)));
+            dict.Add("Window", Color.FromArgb(unchecked((int)0xffffffff)));
+            dict.Add("WindowFrame", Color.FromArgb(unchecked((int)0xff000000)));
+            dict.Add("WindowText", Color.FromArgb(unchecked((int)0xff000000)));
```

Comment:
Since we need this list anyway in order to populate the dictionary, did you consider just making the enum internal instead of public?